tests/php/core-api/wpcom-fields/test-post-fields-publicize-connections.php

```diff
@@ -0,0 +1,283 @@
+
+  private $draft_id = 0;
+  private $_backup_wp_rest_additional_fields;
+
+  public static function wpSetUpBeforeClass( $factory ) {
+    register_post_type( 'example-with', array(
+      'show_in_rest' => true,
+      'supports' => array( 'publicize', 'custom-fields' )
+    ) );
+
+    register_post_type( 'example-without', array(
+      'show_in_rest' => true,
+      'supports' => array( 'publicize' )
+    ) );
+
+    add_post_type_support( 'post', 'publicize' );
+
+    self::$user_id = $factory->user->create( array( 'role' => 'administrator' ) );
+
+    // Hacky :(
+    if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+      self::setup_connections_wpcom();
+    } else {
+      self::setup_connections_jetpack();
+    }
+  }
+
+  static function setup_connections_wpcom() {
+    global $wpdb;
+
+    $wpdb->insert( 'external_access_tokens', array( 'user_id' => self::$user_id, 'external_display' => 'test-display-name456', 'token' => 'fb-token', 'provider' => 'facebook' ) );
+    $keyring_token_id = (string) $wpdb->insert_id;
+    self::$connection_ids[] = $keyring_token_id;
+    $wpdb->insert( 'publicize_connections', array( 'user_id' => self::$user_id, 'token_id' => $keyring_token_id, 'blog_id' => get_current_blog_id() ) );
+
+    $wpdb->insert( 'external_access_tokens', array( 'user_id' => self::$user_id, 'external_display' => 'test-display-name123', 'token' => 't-token', 'provider' => 'tumblr' ) );
+    $keyring_token_id = (string) $wpdb->insert_id;
+    self::$connection_ids[] = $keyring_token_id;
+    $wpdb->insert( 'publicize_connections', array( 'user_id' => 0 /* global connection */, 'token_id' => $keyring_token_id, 'blog_id' => get_current_blog_id() ) );
+  }
+
+  static function setup_connections_jetpack() {
+    Jetpack_Options::update_options( array(
+      'publicize_connections' => array(
+        // Normally connected facebook
+        'facebook' => array(
+          'id_number' => array(
+            'connection_data' => array(
+              'user_id'  => self::$user_id,
+              'token_id' => 'test-unique-id456',
+              'meta'     => array(
+                'display_name' => 'test-display-name456',
+              ),
+            ),
+          ),
+        ),
+        // Globally connected tumblr
+        'tumblr' => array(
+          'id_number' => array(
+            'connection_data' => array(
+              'user_id'  => 0,
+              'token_id' => 'test-unique-id123',
+              'meta'     => array(
+                'display_name' => 'test-display-name123',
+              ),
+            ),
+          ),
+        ),
+      ),
+		) );
+		self::$connection_ids[] = 'test-unique-id456';
+		self::$connection_ids[] = 'test-unique-id123';
+  }
+
+  public static function wpTearDownAfterClass() {
+    unregister_post_type( 'example-with' );
+    unregister_post_type( 'example-without' );
+  }
+
+  public function setUp() {
+    $this->draft_id = $this->factory->post->create( array( 'post_status' => 'draft' ) );
```

Comment:
I think this might be the culprit... That is causing things to be break.

Reply:
going to try to delete these posts in `wp_delete_post`